docs/overview/queries/elastic_query_fuzzy.md

```diff
@@ -0,0 +1,40 @@
+---
+id: elastic_query_fuzzy
+title: "Fuzzy Query"
+---
+
+The `Fuzzy` query returns documents that contain terms similar to the search term, as measured by a Levenshtein edit distance.
```

Comment:
Add a link to this Levenshtein edit distance.
`https://en.wikipedia.org/wiki/Levenshtein_distance`